packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -32,16 +32,29 @@ function BlockGroupToolbar( { label = _x( 'Group', 'verb' ) } ) {
 		[ clientIds ]
 	);
 
-	const onConvertToGroup = () => {
+	const variationAttributes = {
+		row: { type: 'flex' },
+		stack: { type: 'flex', orientation: 'vertical' },
+	};
+
+	const onConvertToGroup = ( variation = 'group' ) => {
 		const newBlocks = switchToBlockType(
 			blocksSelection,
 			groupingBlockName
 		);
-		if ( newBlocks ) {
+
+		if ( newBlocks && newBlocks.length > 0 ) {
+			newBlocks[ 0 ].attributes.layout =
```

Comment:
it would be nice if these attributes could be passed to `switchToBlockType`, or some related method, but I couldn't see a way to do that

Reply:
... maybe updateBlockAttributes would be better here, will take a look at that